app/services/kenui/email_notification_service.rb

```diff
@@ -46,10 +46,13 @@ def set_configuration_per_account(account_id, event_types, user = 'kenui', reaso
 
       def email_notification_plugin_available?(options = nil)
 
-        nodes_info = KillBillClient::Model::NodesInfo.nodes_info(options)
-        plugins_info = nodes_info.first.plugins_info
-        return !(plugins_info.find_index { |plugin| plugin.plugin_key.include?('email-notifications')}).nil?, nil
-      rescue => e
+        # inquire if the plugin is listening
+        path = KILLBILL_EMAIL_NOTIFICATION_PREFIX
+        KillBillClient::API.get path, nil, options
```

Comment:
One suggestion would be to update the servlet in the plugin to do something more meaningful, have a different name and/or at least add a comment in the plugin. Since it's called `hello` and doesn't do much, it could easily be "cleaned up" in the future otherwise! 😼